week-2-storage/README.md

```diff
@@ -0,0 +1,209 @@
+# Storage and processing of ML data
+
+## Deployment and usage `minio` on `kind`
+
+### Local cluster with `kind`
+
+1. Install `kind`
+
+2. Create local cluster and play around with it
+
+```bash
+kind create cluster --name local-cluster
+kubectl cluster-info --context kind-local-cluster
+kubectl get {nodes|pods|deploy|services}
+kind get nodes --name=local-cluster
+>>> local-cluster-control-plane
+kind get clusters
+>>> local-cluster
+kind get kubeconfig --name=local-cluster
+kind delete cluster --name=local-cluster
+```
+
+3. [OPTIONAL] Load your docker image into local cluster (test with well-known web service)
+
+```bash
+kind load docker-image yevhenk10s/prjctr02-infrastructure:latest --name=local-cluster
+
+kubectl apply -f ../week-1/.cluster/pod-conf.yaml
+kubectl port-forward web-dl 8000:5000
+curl localhost:8000
+curl -X POST localhost:8000 -H 'Content-Type: application/json' -d '{"texts": ["hello, world", "this is a sentance"]}'
+kubectl delete -f ../week-1/.cluster/pod-conf.yaml
+```
+
+### Install and initialize `minio`
+
+Main HOWTHO is on the official [README](https://github.com/minio/operator/blob/master/README.md).
+
+1. Install (locally)
+
+```bash
+wget -qO- https://github.com/minio/operator/releases/latest/download/kubectl-minio_linux_amd64.zip | bsdtar -xvf- -C ~/.local/bin
+chmod +x ~/.local/bin/kubectl-minio
+```
+
+2. Verify installation
+
+```bash
+kubectl minio version
+>>> v4.5.0
+```
+3. Creates a new namespace for the MinIO Tenant.
+
+```bash
+kubectl create namespace minio-tenant-1
+```
+
+4. Initialize the **Operator**
+
+```bash
+kubectl minio init
+```
+
+Check initialization status:
+
+```bash
+kubectl get pods -n minio-operator
+kubectl get deploy -n minio-operator
+kubectl get services -n minio-operator
+kubectl get all --namespace minio-operator
+```
+
+Wait for all pods (`console` and `minio-operator`) to start.
+
+```bash
+kubectl get pods -n minio-operator
```

Comment:
wow, have not seen this way of deploying minio: 

I am using this yamls https://github.com/kubernetes/examples/tree/master/staging/storage/minio , should be easier

Reply:
Oh, if I only knew about the way of deployment you show. It would saved me about 16h of struggling with minio and kubernetes.

The only official docs I had found were:
- https://docs.min.io/docs/deploy-minio-on-kubernetes.html
- https://github.com/minio/operator/blob/master/README.md

As you can see there are no description of such simplified and neat way of deployment you've shown.